extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -197,4 +205,64 @@ public boolean isAnyPropertySet() {
             return sql || jdbcWarnings.isPresent();
         }
     }
+
+    @ConfigGroup
+    public static class HibernateOrmConfigCache {
+
+        /**
+         * Cache configuration properties
+         */
+        @ConfigItem(name = ConfigItem.PARENT)
+        public Map<String, CacheConfig> cacheConfig;
+
+        public boolean isAnyPropertySet() {
+            return !cacheConfig.isEmpty();
+        }
+    }
+
+    @ConfigGroup
+    public static class CacheConfig {
+        final static String EXPIRATION_MAX_IDLE = "expiration.max-idle";
+        final static String MEMORY_OBJECT_COUNT = "memory.object-count";
```

Comment:
I don't think it's a good idea to use the same constant, especially since they are totally unrelated. These ones need to be in the `HibernateConfigUtil` class.

Even if they end up having the same value in the end, they are totally different beasts.

A comment pointing to the cache implementation where these constants are declared would be nice.